### PR TITLE
Update lst_creator and Arm64 lstFile

### PR DESCRIPTION
**Pull request by jashook**
Update Arm64 lstFile to include all new 300 failing tests and 1700 test additions. This also updates the test drop for Arm64 test runs.

@RussKeldorph @adiaaida ptal

### CODE REVIEW

**Comment by jashook** [comment]
@dotnet-bot test Windows_NT Arm64 checked

**Comment by jashook** [comment]
Note. I am expected flaky builds of arm64. I will wait to merge until I get consistent clean runs.

File: tests/scripts/lst_creator.py [hunk 1]
```diff
@@ -7,227 +7,373 @@
...
 
-    unique_output = dict()
-    largest_value = 0
+    Returns:
+        test_list ([str]): List of the tests. Note this is defined to be every
+                         : cmd file under the base_dir.
 
-    # If there was an old list file passed. Parse it for all the old tests.
+    Note:
+        To find the tests correctly you must have built the tests correctly and
```
Comment (adiaaida) [inline tests/scripts/lst_creator.py]: "you must build the tests correctly and pass that directory"

**Comment by jashook** [comment]
@dotnet-bot test Windows_NT Arm64 checked

**Comment by jashook** [comment]
@dotnet-bot test Windows_NT Arm64 checked

**Comment by jashook** [comment]
@dotnet-bot test Windows_NT arm64 Cross Debug Build

**Comment by jashook** [comment]
@dotnet-bot test Windows_NT arm64 Cross Debug Build

**Comment by jashook** [comment]
@dotnet-bot test Windows_NT Arm64 checked

**Comment by jashook** [comment]
@dotnet-bot test Windows_NT Arm64 checked

**Comment by jashook** [comment]
@dotnet-bot test Windows_NT Arm64 checked

**Comment by jashook** [comment]
@dotnet-bot test Windows_NT Arm64 checked

**Comment by jashook** [comment]
@dotnet-bot test Windows_NT Arm64 checked

**Comment by jashook** [comment]
@dotnet-bot test Windows_NT Arm64 checked

**Comment by jashook** [comment]
@dotnet-bot test Windows_NT Arm64 checked

**Comment by jashook** [comment]
@dotnet-bot test Windows_NT Arm64 checked

**Comment by jashook** [comment]
@dotnet-bot test Windows_NT Arm64 checked

**Comment by jashook** [comment]
@dotnet-bot test Windows_NT Arm64 checked

**Comment by jashook** [comment]
@dotnet-bot test Windows_NT Arm64 checked

**Comment by jashook** [comment]
@adiaaida I believe everything should be addressed now.

**Pull request merged**